### PR TITLE
Review the alternatives section

### PR DESCRIPTION
**Pull request by jverbraeken**

### CODE REVIEW

File: alternatives.tex [hunk 1]
```diff
@@ -29,14 +28,14 @@ \subsubsection{Alternatives to Distributed machine learning}
 \end{figure}
 
 The most important aspect of the TPU is the Matrix Multiply unit that is positioned
-on the right of the diagram. As can be seen the TPU uses a PCIe bus to communicate
-with a server, so it is not a stand alone unit. Which is usefull as it can then also
-be used in a distributed setting.
+on the right of the diagram. As can be seen, the TPU uses a PCIe bus to communicate
+with a server so it is not a stand-alone unit ()which is useful as it can then also
```
Comment (Kloppie5) [inline alternatives.tex]: ()which?

Comment (Kloppie5) [reply alternatives.tex]: aah, editor autocomplete of the close parenthesis

File: alternatives.tex [hunk 2]
```diff
@@ -29,14 +28,14 @@ \subsubsection{Alternatives to Distributed machine learning}
...
+be used in a distributed setting).
 
-The performance improvement of the TPU vs a regular CPU/GPU combinations is not only
-in the increased processing power, but also in the efficiency, which is important
-for big companies that want to run many TPUs in the same building. In Figure \ref{TPU_Relative_Performance}
-it can be seen that the performance/watt relative to GPUs and CPUs of a TPU is many times higher,
+The performance improvement of the TPU over regular CPU/GPU combinations is not only
+the increased processing power, but also the efficiency, which is important
+for big companies that want want to minimize their energy costs. In Figure \ref{TPU_Relative_Performance},
```
Comment (Kloppie5) [inline alternatives.tex]: want want

File: alternatives.tex [hunk 3]
```diff
@@ -57,24 +56,23 @@ \subsubsection{Alternatives to Distributed machine learning}
...
 of the Epiphany architecture. The Epiphany architecture is a Multiple Instruction,
 Multiple Data (MIMD) architecture that uses an array of processors, that all use the
 same memory to speed up execution of floating point operations\cite{Olof16}.
 
-The newest chip that Adapteva have managed to produce is the Epihpany V, which uses
-1024 cores on a single chip\cite{Olof16}. Although the power consumption of their newest chip
-is not yet public information, they have released numbers boasting a power usage of
-only 2 watt on their Epiphany IV chip\cite{Adap}.
+The newest chip of the major manufacturer Adapteva is the Epihpany V, which uses
+1024 cores on a single chip\cite{Olof16}. Although Adapteva has not publicsed the power consumption of the Epiphany V yet,
```
Comment (Kloppie5) [inline alternatives.tex]: publicsed

File: appendixA.tex [hunk 4]
```diff
@@ -0,0 +1,83 @@
+\section{Appendix A - Research Questions}
```
Comment (Kloppie5) [inline appendixA.tex]: duplicate change; #24

**Review by Kloppie5 — CHANGES_REQUESTED** [review]
Please remove parts that are already the subject of other pull requests (Appendix A)

**Review by Kloppie5 — APPROVED** [review]
I'm confused why the earlier parts are writing continuously and later parts are broken into several lines

**Comment by jverbraeken** [comment]
I prefer writing everything continuously because then you don't have to use so many enters, but was getting a bit tired from removing all those enters at the end

**Pull request merged**